src/webots/vrml/WbProtoManager.cpp

```diff
@@ -23,6 +23,7 @@
 #include "WbMultipleValue.hpp"
 #include "WbNetwork.hpp"
 #include "WbNode.hpp"
+#include "WbNodeOperations.hpp"
```

Comment:
Note that here you are adding an additional circular dependency that we will need to remove (see #4981 ).